src/summary/history/conditions-card.component.tsx

```diff
@@ -66,15 +67,9 @@ export default function ConditionsCard(props: ConditionsCardProps) {
             </SummaryCardRow>
           );
         })}
-      <div className={style.conditionMore}>
-        <svg className="omrs-icon">
-          <use
-            xlinkHref="#omrs-icon-chevron-down"
-            fill="var(--omrs-color-ink-low-contrast)"
-          ></use>
-        </svg>
-        <p>more</p>
-      </div>
+      <SummaryCardFooter linkTo={`/patient/${patientUuid}/chart/conditions`}>
```

Comment:
Great job. Just curious, won't `/notes` just work the same way as using `/patient/patientUuid/chart/notes`?

Reply:
`/notes` resolves to `https://openmrs-spa.org/openmrs/spa/notes` which is not the correct URL. 
Therefore changed it to resolve to the correct URL `https://openmrs-spa.org/openmrs/spa/patient/90f7f0b4-06a8-4a97-9678-e7a977f4b518/chart/notes`